app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -1114,6 +1114,11 @@ public void crowdTranslate(MenuItem x) {
        // startActivity(new Intent(this, LanguageEditor.class).setFlags(Intent.FLAG_ACTIVITY_NEW_TASK));
         startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://crowdin.com/project/xdrip")).addFlags(Intent.FLAG_ACTIVITY_NEW_TASK));
     }
+    
+    public void documentationSupport(MenuItem x) {
+        // startActivity(new Intent(this, LanguageEditor.class).setFlags(Intent.FLAG_ACTIVITY_NEW_TASK));
+        startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://navid200.github.io/xDrip/")).addFlags(Intent.FLAG_ACTIVITY_NEW_TASK));
```

Comment:
As parapenT1sta mentioned, https://xdrip.readthedocs.io fits better because Navids pages are more like for faq/support.

Reply:
Great! Love the hyperlink to the official documentation. I will do another PR to xDrip documentation to include a link in the troubleshoot menu section to site of Navid. Currently both sites are a valuable resource, which - unfortunately - are separate websites.

Reply:
@petervanrijt Thanks for the contribution.  
  
> unfortunately - are separate websites

Those two being two different websites is a strength in my opinion, not a weakness.
Patrick and I work closely and provide feedback and suggestions to each other.
We constantly tag each other when there is a question that the other person can better handle.

I can ensure accuracy by limiting the material on my site to what I have expertise in.  Same goes I believe for Patrick.

We don't even have links to those sites on the wiki.  In my opinion, we should.